test/import-cmd.test.js

```diff
@@ -19,7 +19,7 @@ import { fetch } from '../src/fetch-utils.js';
 import ImportCommand from '../src/import.cmd.js';
 
 const TEST_DIR = path.resolve(__rootdir, 'test', 'fixtures', 'import');
-const SAMPLE_HOST = 'https://www.sample.com';
+const SAMPLE_HOST = 'http://www.sample.com';
```

Comment:
switched to `http` because that site will cause a ECONNRESET in the ALPN negotiation phase, which nock doesn't override.